### ISSUE TITLE
Values don't refresh/reflect external changes when externally modified.

### ISSUE THREAD

**Issue by rokejulianlockhart** [MuntashirAkon/SetEdit#21]
If I increase the brightness via the slider that System UI provides, `screen_brightness` does not change until I switch tables and return.

**Comment by rokejulianlockhart** [comment]
@MuntashirAkon, perhaps the preferences being displayed to the screen (to avoid using unnecessary CPU) are automatically refreshed every few milliseconds? That's what most task managers, at least the few designed to display thousands of processes, do.

**Comment by drogga** [comment]
Yes, it would be really nice if they dynamically refreshed - Live...

BTW, I "monitor" the changes using an app called `Settings Watcher` (by JRoot3D.dev - it's on PS), so I know what's going on when I need\have to ;)

**Comment by MuntashirAkon** [comment]
> _solely the preferences being displayed to the screen_ (to avoid using unnecessary CPU) are automatically refreshed every few milliseconds?

Hmm, that's a good point. I'll think about it. I think it's not difficult to update the contents that way. But consistency issues may arise which needs to be handled.
